.github/workflows/notebooks.yml

```diff
@@ -28,16 +28,7 @@ jobs:
         with:
           virtualenvs-create: true
           virtualenvs-in-project: true
-
-      - name: Load cached venv
-        id: cached-poetry-dependencies
-        uses: actions/cache@v2
-        with:
-          path: .venv
-          key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
-
       - name: Install dependencies
-        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
         run: |
           poetry install -E all -vv
```

Comment:
May be the same move with installing jupyter and release here?

Reply:
Fixed